kimchi/src/error.rs

```diff
@@ -67,6 +67,9 @@ pub enum VerifyError {
 
     #[error("the evaluation for {0:?} is missing")]
     MissingEvaluation(crate::circuits::expr::Column),
+
+    #[error("the commitment for {0:?} is missing")]
```

Comment:
I think we should have the error also answer "missing from what?" (same comment for the error above)